### PR TITLE
Add docs for integrations-level outputs

### PR DESCRIPTION
**Pull request by kilfoyle**
This adds a [Set integration-level outputs](https://ingest-docs_bk_1218.docs-preview.app.elstc.co/guide/en/fleet/master/integration-level-outputs.html) page into the "Manage integrations" section of the Fleet & Elastic Agent Guide. There's also a prominent link to that section from the Fleet [Output settings](https://ingest-docs_bk_1218.docs-preview.app.elstc.co/guide/en/fleet/master/fleet-settings.html#output-settings) page. 

@jen-huang please let me know if I've missed anything. Also, I assume this feature will land in 8.16 but correct me if I've got that wrong. I'll wait for your code PR to merge before I merge this one.

Closes: #1217

### CODE REVIEW

File: docs/en/ingest-management/integrations/integration-level-outputs.asciidoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ This includes the case where an integration policy belongs to multiple {agent} p
 [discrete]
 == Configure the output for an integration policy
 
-To conifigure an integration-level output:
+To conifigure an integration-level output for {agent} data:
```
Comment (karenzone) [inline docs/en/ingest-management/integrations/integration-level-outputs.asciidoc:24]:
```suggestion
To configure an integration-level output for {agent} data:
```

**Review by jen-huang — APPROVED** [review]
Information looks accurate to me. And yes this will be for 8.16. Thank you!

File: docs/en/ingest-management/integrations/integration-level-outputs.asciidoc (hunk 1 above)
Comment (kilfoyle) [reply docs/en/ingest-management/integrations/integration-level-outputs.asciidoc:24]: Thanks for catching that @karenzone!

**Review by karenzone — APPROVED** [review]
LGTM

**Pull request merged**